src/engine/gfx/sprite_animations.asm

```diff
@@ -137,9 +137,9 @@ GetSpriteAnimCounter:
 	pop hl
 	ret
 
-_HandleAllSpriteAnimations:
+_HandleAllSpriteAnimations::
 	push af
-	ld a, [wd5d7] ; skip animating this frame if enabled
+	ld a, [wAllSpriteAnimationsDisabled] ; skip animating this frame if enabled
```

Comment:
heh, this comment is a bit misleading now, since by "if enabled" it means if the disabled flag is set.